api/src/main/java/org/openmrs/ConditionVerificationStatus.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs;
+
+/**
+ * The ConditionVerificationStatus is what we've historically called "diagnosis certainty" and what FHIR refers to as
+ * ConditionVerificationStatus. This is the verification status to support or decline the clinical status of the condition or
+ * diagnosis. The following subset of FHIR statuses are currently defined:
+ * <li>{@link #PROVISIONAL}</li>
+ * <li>{@link #CONFIRMED}</li>
+ */
+public enum ConditionVerificationStatus {
+	
+	/**
+	 * This is a tentative diagnosis - still a candidate that is under consideration. This Was called "Presumed" in the 
```

Comment:
"This Was called". Should it be Was or was? 😊

Reply:
:sweat_smile: Opps! I totally missed that one.

Reply:
@dkayiwa i have Updated the English as it should be.